tests/TestCase/View/Helper/RecaptchaHelperTest.php

```diff
@@ -11,10 +11,12 @@
 class RecaptchaHelperTest extends TestCase
 {
 
-    public function setUp()
+    public function setUp(): void
     {
         parent::setUp();
 
+        $this->loadPlugins(['Recaptcha']);
```

Comment:
I don't think this is necessary. Just the autoloading provided by composer should be enough.

Reply:
Without it, it does not find the `templates/element/recaptcha.php` element

Reply:
Ah, I overlooked that we need to use templates too which are not autoloaded :slightly_smiling_face:.